docs/topics/scheduler.rst

```diff
@@ -0,0 +1,35 @@
+.. _topics-scheduler:
+
+=========
+Scheduler
+=========
+
+.. module:: scrapy.core.scheduler
+
+The scheduler component receives requests from the :ref:`engine <component-engine>`
+and stores them into persistent and/or non-persistent data structures.
+It also gets those requests and feeds them back to the engine when it
+asks for a next request to be downloaded.
+
+
+Overriding the default scheduler
+================================
+
+You can use your own custom scheduler class by supplying its full
+Python path in the :setting:`SCHEDULER` setting.
+
+
+Minimal scheduler interface
+===========================
+
+.. autoclass:: BaseScheduler
+   :members:
+
+
+Default Scrapy scheduler
+========================
+
+.. autoclass:: Scheduler
+   :members:
+
+   .. automethod:: __len__
```

Comment:
Is not much more succinct, but it seems like [this](https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#directive-option-automodule-special-members) is “the way”:

```suggestion
   :members:
   :special-members: __len__
```